engines/e_dasync.c

```diff
@@ -803,3 +787,125 @@ static int dasync_aes128_cbc_hmac_sha1_cleanup(EVP_CIPHER_CTX *ctx)
      */
     return dasync_cipher_cleanup_helper(ctx, EVP_aes_128_cbc_hmac_sha1());
 }
+
+
+/*
+ * RSA implementation
+ */
```

Comment:
Really, *really* nice refactoring!  I know I was pushing hard when you tried to disable all engines here, so this turnaround gets the corresponding praise.  Well done!

Reply:
It was when I switched to only this engine being disabled that I had difficulties.  It's a demo, nobody in their right mind uses it, why??? :)

Still, it is done now.

Reply:
If nothing else, we gain experience. I don't think that should be underestimated